data_prep/graph_utils.py

```diff
@@ -71,7 +71,7 @@ def word_pair_frequency(windows):
     return word_pair_freq
 
 
-def pmi(corpus, window_size=15):
+def PMI(corpus, window_size=15):
```

Comment:
rip https://www.python.org/dev/peps/pep-0008/#function-and-variable-names :(

Reply:
it didn't match with the other files anymore...

Reply:
Hm that's a problem, but then the other files should be fixed instead

Reply:
Fixed it